src/parsing/scope.ml

```diff
@@ -647,7 +647,7 @@ let scope_rule :
     fatal p_lhs.pos
       "Symbol %s has been declared constant, it cannot be used as the \
        head of a rewrite rule LHS." pr_sym.sym_name;
-  if pr_sym.sym_opaq || Timed.(!(pr_sym.sym_def)) <> None then
+  if Timed.(!(pr_sym.sym_opaq)) || Timed.(!(pr_sym.sym_def)) <> None then
     fatal p_lhs.pos "No rewriting rule can be given on a defined symbol.";
```

Comment:
"No rewriting rule can be added on an opaque symbol or a symbol already defined with ≔'